.github/workflows/release-notes-linter.yaml

```diff
@@ -0,0 +1,32 @@
+---
+name: Release Note Linter
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Release Note Linter
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install wget
+      run: sudo apt-get install wget
+    - name: Install jq
+      run: sudo apt-get install jq
+    - name: Sanity Check Release Notes
+      env:
+        PR_NUMBER: ${{ github.event.number }}
+      run: |
+        # Validate PR release notes
+        echo "Going to validate PR ${PR_NUMBER}"
+
+        MATCHES=$(wget -q  -O- https://api.github.com/repos/shipwright-io/build/pulls/${PR_NUMBER} | jq '.body | match("(```release-note\r\n(.*|NONE|action required: .*)\r\n```)")')
```

Comment:
technically speaking not the same arcane bash I derived from tekton and landed with at https://github.com/shipwright-io/build/blob/b56b30f358908a866874ce7ddadcb463c4c748d9/.github/draft_release_notes.sh#L81-L82

but to be fair a) I too diverged slightly from tekton when I constructed this to conform to the slight differences we had in out process, and b) you have added the use of `jq` that I removed, and c) they have tweaked their overall script recently, though git blame says the arcane wget/jq/grep magic has not changed for them in 9 months

for reference https://github.com/tektoncd/plumbing/blob/9c93fcf9b764ef4fcb52f9aa34f7f2551d2fe804/tekton/resources/release/base/github_release.yaml#L133-L135

and there has been talk of switching all this from bash to golang if we get too fed up with things :-)

long winded build up to:

"Unless we encounter discrepancies when cutting release notes next time, and if this passes PR tests, let's go with this , and only iterate further if needed" :-)

/lgtm